maintainers/maintainer-list.nix

```diff
@@ -4326,6 +4326,11 @@
     githubId = 14034137;
     name = "Mostly Void";
   };
+  ditsuke = {
+    name = "Tushar";
+      email = "hello@ditsuke.com";
+      github = "ditsuke";
```

Comment:
Can you align this please? Also I think it needs your githubid. At the top off the file is described how to obtain it.